raiden/transfer/channel.py

```diff
@@ -126,23 +130,30 @@ class UnlockGain(NamedTuple):
     from_partner_locks: TokenAmount
 
 
-def get_sender_expiration_threshold(lock: LockType) -> BlockNumber:
+def get_initial_lock_expiration(
+    block_number: BlockNumber, reveal_timeout: BlockTimeout
+) -> BlockExpiration:
+    """ Returns the expiration used for all hash-time-locks in transfer. """
```

Comment:
this is not correct, this is the upper bound, and the value used for the initiator. It may not be the value used by all.